.github/workflows/release.yml

```diff
@@ -116,14 +126,14 @@ jobs:
             !**/node_modules
             **/tsdoc.json
             **/stories.js
-          key: ${{ runner.os }}-build-cache-${{ hashFiles('package.json', 'yarn.lock', '**/src/', '!**/node_modules', '!**/dist') }}
+          key: ${{ runner.os }}-build-cache-${{ hashFiles('package.json', 'pnpm-lock.yaml', '**/src/', '!**/node_modules', '!**/dist') }}
 
       - name: Version packages
         id: changesets
         uses: changesets/action@v1
         with:
-          version: yarn changeset version
-          publish: yarn run publish
+          version: pnpm changeset version
+          publish: pnpm run publish
```

Comment:
This should be `pnpm publish -r` 
https://pnpm.io/using-changesets#releasing-changes